test/Test/Database/LSMTree/StateMachine.hs

```diff
@@ -2248,10 +2248,10 @@ data Stats = Stats {
     -- === Final tags (per action sequence, per table)
     -- | Number of actions per table (succesful or failing)
   , numActionsPerTable :: !(Map Model.TableID Int)
-    -- | The size of tables that were closed. This is used to augment the table
-    -- sizes from the final model state (which of course has only tables still
-    -- open in the final state).
-  , closedTableSizes   :: !(Map Model.TableID Int)
+    -- | The state of model tables at the point they were closed. This is used
+    -- to augment the tables from the final model state (which of course has
+    -- only tables still open in the final state).
+  , closedTables       :: !(Map Model.TableID Model.SomeTable)
```

Comment:
It turned out that I never needed this change (as I had expected I would) but I've left it here anyway since it's a cheap and sane generalisation, and may indeed be useful later.